modules/tile-converter/src/3d-tiles-converter/3d-tiles-converter.ts

```diff
@@ -96,7 +98,18 @@ export default class Tiles3DConverter {
     this.geoidHeightModel = await load(egmFilePath, PGMLoader);
     console.log('Loading egm file completed!'); // eslint-disable-line
 
-    this.sourceTileset = await load(inputUrl, I3SLoader, this.loaderOptions);
+    this.slpkFilesystem = await openSlpk(inputUrl);
+
+    this.sourceTileset = await loadFromArchive(
```

Comment:
So I guess this could be treated as a `Source` in the architecture we were discussing this morning.